src/mesh/PhoneAPI.cpp

```diff
@@ -451,7 +448,7 @@ size_t PhoneAPI::getFromRadio(uint8_t *buf)
 
 void PhoneAPI::sendConfigComplete()
 {
-    LOG_INFO("getFromRadio=STATE_SEND_COMPLETE_ID");
+    LOG_INFO("Config Send Complete.");
     fromRadioScratch.which_payload_variant = meshtastic_FromRadio_config_complete_id_tag;
```

Comment:
I would change all intermediate steps to LOG_DEBUG till the Config Send Complete is issued. Start and stop of config transfer is an info.